api-cli/src/commands/start.ts

```diff
@@ -98,10 +106,12 @@ export default class Start extends Command {
     }
 
     //inbound proxy
-    const target = processSetting(config.proxy.target, inputs)
+    const proxyTarget = flags.target ? flags.target : config.proxy.target
+    const proxyPort = flags.port ? parseInt(flags.port, 10) : config.proxy.port
+    const target = processSetting(proxyTarget, inputs)
     await proxySession.start({
       target,
-      port: config.proxy.port
+      port: proxyPort
     })
     this.log(fromOptic(`Starting ${colors.bold(config.name)} on Port: ${colors.bold(config.proxy.port.toString())}, with ${colors.bold(config.commands.start)}`))
```

Comment:
Hey @esopian  -- can you update the log statement to use the new variables?

Reply:
@acunniffe missed this comment.  You've probably moved passed this but I went ahead and changed anyway